tests/test___main__.py

```diff
@@ -28,10 +28,10 @@
 
 class Test_CLI_exit_code:
```

Comment:
[consider] Worth a test that omitting `-g` flags an error.

Reply:
Added tests that test for the error string for ommited `-g` and `-o` flags in `stderr`